bin/vitamin.js

```diff
@@ -147,8 +146,8 @@ const build = (options, hotCallback, restartServer) => (options.hot ?
 
 
 const test = (args) => {
-    npmRunSpawn(
-        'jest',
+    spawn(
+        vitaminResolve('node_modules', '.bin', 'jest'),
```

Comment:
- `yarn` should not be required to use any part of `vitamin`.
- and i agree with you @christophehurpeau, seems better to use directly vitamin's `node_modules/.bin/jest` since we ship with it. _(i don't know why yet, but it seems to have some drawbacks_